spec/unit/deployment/config/loader_spec.rb

```diff
@@ -0,0 +1,26 @@
+require 'spec_helper'
+require 'r10k/deployment/config/loader'
+
+describe R10K::Deployment::Config::Loader do
+
+  it "creates R10K::Deployment::Config::Loader instances" do
```

Comment:
In some cases we may need to test the object type returned by `.new`, specifically when we're overloading new to [redirect a deprecated class](https://github.com/puppetlabs/r10k/blob/master/lib/r10k/deployment/source.rb#L33). But unless that's happening, we can generally assume that `.new` is behaving as expected and skip testing it. If we're not getting back a `R10K::Deployment::Config::Loader` object, either all the tests pass, the desired behavior is still occurring, and there's not a problem, or the tests crash and burn, in which case this test would fail as well. If this covers behavior that wouldn't be caught otherwise then we should keep it; if another test will fail if another test fails then we can omit it.